### PR TITLE
feat(modules/lb_http_ext_global): Added ability to add custom request headers

### PR DESCRIPTION
**Pull request by ntwrkguru**
## Description

Added ability to support Custom request headers, e.g. X-Forwarded-For to the `lb_http_ext_global` module.

## Motivation and Context
Client requested the ability to manage the XFF controls in code

## How Has This Been Tested?

Locally

## Types of changes

- New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes if appropriate.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by ntwrkguru** [comment]
Fixes #188

File: modules/lb_http_ext_global/main.tf [hunk 1]
```diff
@@ -57,10 +57,11 @@ resource "google_compute_url_map" "default" {
...
 resource "google_compute_backend_service" "default" {
-  name        = var.name
-  port_name   = var.backend_port_name
-  protocol    = var.backend_protocol
-  timeout_sec = var.timeout_sec
+  name                   = var.name
+  port_name              = var.backend_port_name
+  protocol               = var.backend_protocol
+  custom_request_headers = var.custom_request_headers
+  timeout_sec            = var.timeout_sec
```
Comment (michalbil) [inline modules/lb_http_ext_global/main.tf:64]:
```suggestion
  timeout_sec            = var.timeout_sec
  
```

File: modules/lb_http_ext_global/variables.tf [hunk 2]
```diff
@@ -87,6 +87,11 @@ variable "http_forward" {
   default     = true
 }
 
+variable "custom_request_headers" {
+  type        = list(string)
+  default     = []
+  description = "(Optional) Headers that the HTTP/S load balancer should add to proxied responses."
+}
```
Comment (michalbil) [inline modules/lb_http_ext_global/variables.tf:94]:
```suggestion
}

```

**Review by michalbil — APPROVED** [review]
👍

**Pull request merged**